src/components/ContactInfo/ContactInfo.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+
+const ContactInfo = ({ body, title }) => {
+  let contact = body;
+
+  return (
+    <div className="contact_info">
```

Comment:
This is the more consistent way we are doing style names, please update accordingly

```suggestion
    <div className="contact-info">
```